inst/rmarkdown/templates/fmri_prereg/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,662 @@
+    institution : Konstanz Business School
+
+output: prereg::fmri_prereg
+---
+
+
+<!-- Goal of extended fMRI preregistration template**
+
+The goal of this template is to provide sufficient information in preregistration for fMRI data design to increase reproducible reporting practices. 
+
+Text was adopted from the OSF preregistration challenge template to include the details important for fMRI psychology design. OSF template is retained in some areas and edited in other areas to incorporate both prior published templates and guidelines (Nichols et al., 2016; Poldrack et al., 2008; van't Veer & Giner-Sorolla, 2016). 
+
+https://www.biorxiv.org/content/early/2016/05/20/054262 
+https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2287206/
+https://www.sciencedirect.com/science/article/pii/S0022103116301925 
+
+Quick notes:
+All prompts and tables can be used to fill in as it, or as a checklist of information you include for text for that section or checklist for attached files (e.g., json file of scan parameters). 
+
+**Tip: Using it as a checklist will allow you to write these sections as they will appear in your future paper.**
+
+If a certain table/section does not apply, can just state "N/A." 
+
+This is also a base template and will not be sufficient for every type of fMRI analysis. If your methods require additional checklists, please include! -->
+
+
+## Working Title
+
+Enter your response here.
+
+
+## Introduction
+<!-- This can be as complete as you like, but at least a brief introduction is suggested to help to ensure literature review is done prior to hypotheses and help ensure your hypotheses are properly informed based on the literature.  -->
+
+Enter your response here.
+
+
+## Current Study
+<!-- Any level of information you want to include about the gap your study is addressing. -->
+
+Enter your response here.
+
+
+**Aims & Hypotheses**
+
+| **Essential** | **Checklist** |
+| ----------- | ----------- |
+| Hypotheses include direction of expected results |    ___ |
+| Interactions describe expected shape |     ___ or NA | 
+| Maniputated variables include manipulation checks or explain why not |    ___ or NA | 
+
+| **Recommended** | **Checklist** |
+| ----------- | ----------- |
+| Figure or table to describe expected results |    ___ |
+| Rationals or frameworks included for why certain hypotheses are being tested |     ___ | 
+| Which outcome would be predicted by which theory |     ___ or NA | 
+
+<!-- Adapted from van't Veer & Giner-Sorolla, 2016 -->
+
+## Exisiting Data
+<!-- Preregistration is designed to make clear the distinction between confirmatory tests, specified prior to seeing the data, and exploratory analyses conducted after observing the data. Therefore, creating a research plan in which existing data will be used presents unique challenges. Please select the description that best describes your situation. -->
+
+- Registration prior to creation of data
+
+-	Registration prior to any human observation of the data
```

Comment:
Could you remove the white space here so that it becomes `- Registration prior to any human observation of the data`?